server.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
I've written a installing script at "scripts/install.gs" which can be executed with [gosl](https://github.com/daviddengcn/gosl)